src/listpack.c

```diff
@@ -2048,6 +2198,82 @@ int listpackTest(int argc, char *argv[], int flags) {
         zfree(lp3);
     }
 
+    TEST("lpNextRandom normal usage") {
```

Comment:
FYI. the two below tests both missing lpFree, fix it in #11492

Reply:
:facepalm: Thanks for fixing it!